include/sycl_ext_complex.hpp

```diff
@@ -558,6 +590,26 @@ class complex<_Tp, typename std::enable_if<is_genfloat<_Tp>::value>::type> {
     return __t;
   }
 
+#if defined(_SYCL_EXT_CPLX_FAST_MATH) &&                                       \
+    defined(_SYCL_EXT_CPLX_FAST_MATH_UNSAFE)
```

Comment:
What is this `_SYCL_EXT_CPLX_FAST_MATH_UNSAFE` ? I this we can only fast math. And fast math by definition is unsafe